src/api/types/nes.types.ts

```diff
@@ -1,44 +1,50 @@
-export type ScanInput = { components: string[]; type: 'SBOM' } | { type: 'OTHER' };
+/**
+ * Input parameters for the EOL scan operation
+ */
+export interface InsightsEolScanInput {
+  scanId?: string;
```

Comment:
```suggestion
  scanId?: string;
```

is scanId really optional or can be undefined?

Reply:
It is defined as optional in the monorepo gql:

```
input InsightsEolScanInput {
  scanId: String // optional
  components: [String]! // required - due to the `!`
```